### PR TITLE
Uk/registration state display fix

### PR DESCRIPTION
**Pull request by KeirOsborn**
Hi @lin-d-hop, @Matt-Yorkley and @mkllnk,

New pull request for issue #1186 replacing pull request #1242 

- Moved this over to a new branch called Uk/registration state display fix so that an automatic build is created on buildkite. 

- Removed unused modification to config/locales/en.yml. This: "registration_detail_state_label: "abbr"".

- Removed unintended alteration to Gemfile.lock.

@mkllnk I've rebased this branch against master but I can't work out how to squash my commits into one coherent commit message (i'm guessing that's the rationale behind squashing commits). I had hoped this would happen as a result of rebasing but clearly there's something else that I need to do. Could you point me in the right direction?

Was there anything else that needs working on for this issue?

Thanks,

### CODE REVIEW

**Comment by mkllnk** [comment]
@KeirOsborn If you use the Git command line tool, you can add `-i` for interactive to the rebase command:

```sh
git rebase -i origin/master
```

You can then decide which commits do squash together. You can also change the order, the commit messages, everything. A good tutorial is: https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history

**Comment by KeirOsborn** [comment]
@mklink Cool thanks. Done.

File: Gemfile.lock [hunk 1]
```diff
@@ -732,9 +732,3 @@ DEPENDENCIES
   whenever
   wicked_pdf
   wkhtmltopdf-binary
-
-RUBY VERSION
-   ruby 2.1.5p273
-
-BUNDLED WITH
-   1.13.6
```
Comment (enricostano) [inline Gemfile.lock]: IMHO would be safer to keep `Gemfile.lock` as it is and make sure two things:

 - that every contributor uses the latest `bundler` version
 - that every contributor uses the exactly same ruby version that we have in production
 - that every instance of OFN uses the same ruby version (most difficult part IMHO)

This strategy would avoid future headaches and make deploy process more consistent and stable.

File: app/serializers/api/state_serializer.rb [hunk 2]
```diff
@@ -1,7 +1,12 @@
 class Api::StateSerializer < ActiveModel::Serializer
-  attributes :id, :name, :abbr
+  attributes :id, :name, :abbr, :label
 
   def abbr
     object.abbr.upcase
   end
+
+  def label	
+	eval Spree::Config[:state_display]
```
Comment (enricostano) [inline app/serializers/api/state_serializer.rb]: Identation should be 2 spaces.

Comment (KeirOsborn) [inline app/serializers/api/state_serializer.rb]: Ah cool. Not sure how that happened. Fixed and re-rebased.

**Comment by lin-d-hop** [comment]
Closing as being picked up by @sauloperez

**Pull request closed without merging**